server/main.lua

```diff
@@ -14,7 +14,7 @@ local function createEntity(query)
         query.citizenId,
         query.model,
         joaat(query.model),
-        query.mods or '{}',
+        json.encode(query.mods) or nil,
```

Comment:
json.encode() will not return nil, so the or nil will never occur. 
```suggestion
        query.mods and json.encode(query.mods) or nil,
```